backend/app/views/spree/admin/style_guide/topics/tables/_building_tables.html.erb

```diff
@@ -1,7 +1,19 @@
 <%- snippet = capture do %>
-  <table>
+  <table class='selectable-table'>
     <thead>
+      <tr class="selectable-header no-item-selected">
+        <th><%= check_box_tag 'select-all' %></th>
+        <th colspan="7">
+          <span class="all-selected">
+            All Item selected
+          </span>
+          <span class="custom-selection">
+            <span class="label"></span> Selected Item
```

Comment:
Why `label`? Do you think we can use a better name for this class to describe its content?

Reply:
`selected-item-label` ?

Reply:
What will be the content of that span? It's a number right? `selected-item-count`?

Reply:
You are right, it's a number! I'm going to change it